modules/installation/proc-configuring-an-rhdh-instance-with-tls-in-kubernetes.adoc

```diff
@@ -0,0 +1,104 @@
+[id="proc-configuring-an-rhdh-instance-with-tls-in-kubernetes_{context}"]
+= Configuring an {product-very-short} instance with a TLS connection in Kubernetes
+
+You can configure an {product-very-short} instance with a Transport Layer Security (TLS) connection in a Kubernetes cluster, such as an Azure Red Hat OpenShift (ARO) cluster, any other cloud provider cluster, or your own cluster with proper configuration. However, You must use a public Certificate Authority (CA)-signed certificate to configure your Kubernetes cluster.
```

Comment:
Can we say "any cluster from a supported cloud provider"?  We don't support everything